Dockerfile.test

```diff
@@ -0,0 +1,20 @@
+# docker build -t lsr-test -f Dockerfile.test .
```

Comment:
Why don't you invoke guard via docker-compose?

https://github.com/mtsmfm/language_server-ruby/tree/ab8998db3e14a414ef51130611a743c39978db8f#run-test

We don't have to create new dockerfile

Reply:
Eh.  
Composition just feels heavy to me.  And not all platforms have docker-compose by default.

Which compose file do you suggest I use, and how would I invoke it?

Reply:
If you're using Docker for Mac, docker-compose is already installed.

You can run any commands via docker-compose.
So in this case, `docker-compose run app bundle exec guard` is what you want.
If you'd like to use old ruby, run `docker-compose run ruby-2-3 bundle exec guard`.

There are three docker-compose related files:

- [docker-compose.yml](https://github.com/mtsmfm/language_server-ruby/blob/ab8998db3e14a414ef51130611a743c39978db8f/docker-compose.yml)
- [docker-compose.override.yml](https://github.com/mtsmfm/language_server-ruby/blob/ab8998db3e14a414ef51130611a743c39978db8f/docker-compose.override.yml)
- [docker-compose.ci.yml](https://github.com/mtsmfm/language_server-ruby/blob/ab8998db3e14a414ef51130611a743c39978db8f/docker-compose.ci.yml)

docker-compose will use docker-compose.yml and docker-comopse.override.yml automatically.
docker-compose.ci.yml is only for CI and it isn't used on development env.